### PR TITLE
Vxscan vvsg ballot counter

### PR DESCRIPTION
**Pull request by kofi-q**
## Overview

Related to: https://github.com/votingworks/vxsuite/issues/3185
Closes: https://github.com/votingworks/vxsuite/issues/3059

Relevant changes in first commit (4b182f0455dce82d1575d9076784011e9bfd0175)
Second commit has test snapshot updates for VxMark (can't figure out why those changed).

- Added `BigMetric` component in libs/ui (builds on `LabelledText`) to use for the `ScannedBallotCount` component in VxScan
- Updated the base VxScan layout to persist the ballot count across all screens, per https://github.com/votingworks/vxsuite/issues/3059

## Demo Video or Screenshot

https://user-images.githubusercontent.com/264902/230153613-86eeeb00-668d-4369-832d-5deea89e2f1c.mov

### In context in VxScan:
Screenshot 2023-04-04 at 17 22 10

## Testing Plan
- Added unit test for `BigMetric`
- Updated tests that indirectly render the base layout component to mock out the API call for scanner status

## Checklist

- [n/a] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/ui/src/big_metric.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  const formattedValue = format.count(value);
+  const formattedLabel = `${label}: ${formattedValue}`;
+
+  return (
+    <StyledContainer>
+      <LabelledText aria-hidden label={label}>
+        <StyledValue
+          aria-label={formattedLabel}
+          data-testid={valueElementTestId}
```
Comment (eventualbuddha) [inline libs/ui/src/big_metric.tsx:50]: Is there a way to make `<BigMetric data-testid="…"` do what we want? Or is this the better way to do this?

File: libs/ui/src/big_metric.test.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  screen.getByText('Ballots Scanned');
+
+  // Verify counts are rendered as formatted numbers with accessible labels:
+  expect(screen.getByText('2')).toHaveAccessibleName('Number of Machines: 2');
+  expect(screen.getByText('4,096')).toHaveAccessibleName(
+    'Ballots Scanned: 4,096'
+  );
+});
+
+// To pass satisfy coverage requirements:
```
Comment (eventualbuddha) [inline libs/ui/src/big_metric.test.tsx:21]:
```suggestion
// To satisfy coverage requirements:
```

File: libs/ui/src/big_metric.tsx (hunk 1 above)
Comment (kofi-q) [reply libs/ui/src/big_metric.tsx:50]: Yeah, I believe that would work - I was avoiding having to update the test assertions to include the label text as well, but I'm down to do that folks feel strongly

File: libs/ui/src/big_metric.test.tsx (hunk 2 above)
Comment (kofi-q) [reply libs/ui/src/big_metric.test.tsx:21]: thanks!

File: libs/ui/src/big_metric.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+ */
+export function BigMetric(props: BigMetricProps): JSX.Element {
+  const { label, value, valueElementTestId } = props;
+
+  const formattedValue = format.count(value);
+  const formattedLabel = `${label}: ${formattedValue}`;
+
+  return (
+    <StyledContainer>
+      <LabelledText aria-hidden label={label}>
```
Comment (kshen0) [inline libs/ui/src/big_metric.tsx:47]: Why do we want this to be `aria-hidden`? Based on the `aria-label` on the child element, it seems like we want this tree to be rendered on accessibility devices.

Comment (kofi-q) [reply libs/ui/src/big_metric.tsx:47]: Oh, good catch! I meant to put that on the label, not the whole component - will fix that up

**Review by kshen0 — APPROVED** [review]
LGTM!

**Pull request merged**